my_site/tel_book/views/contact_views.py

```diff
@@ -2,20 +2,20 @@
 from ..models import Contact
 from ..forms import ContactForm
 from django.contrib import messages
+from django.contrib.auth.decorators import login_required
 
-
+# Домашня сторінка
```

Comment:
You don't need such comments and below comments are also unnecessary, but if you want to remain them, please use English language